pom.xml

```diff
@@ -1,71 +1,86 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-	<groupId>datadog</groupId>
-	<artifactId>jmxfetch</artifactId>
-	<version>0.20.2</version>
-	<packaging>jar</packaging>
+    <groupId>datadog</groupId>
+    <artifactId>jmxfetch</artifactId>
+    <version>0.20.3</version>
```

Comment:
We'll bump that in another PR (and probably increase the minor since it's a pretty big change 😛 )

Reply:
Should be fixed now.